udata/core/topic/factories.py

```diff
@@ -2,10 +2,39 @@
 
 from udata import utils
 from udata.core.dataset.factories import DatasetFactory
-from udata.core.reuse.factories import VisibleReuseFactory
+from udata.core.reuse.factories import ReuseFactory
 from udata.factories import ModelFactory
 
-from .models import Topic
+from .models import Topic, TopicElement
+
+
+class TopicElementFactory(ModelFactory):
+    class Meta:
+        model = TopicElement
+
+    title = factory.Faker("sentence")
+    description = factory.Faker("text")
+
+    @classmethod
+    def element_as_payload(cls, elt) -> dict:
+        return {
+            "element": {"id": str(elt["element"].id), "class": elt["element"].__class__.__name__},
+            "title": elt["title"],
+            "description": elt["description"],
+        }
+
+    @classmethod
+    def as_payload(cls) -> dict:
+        elt = cls.as_dict()
+        return cls.element_as_payload(elt)
+
+
+class TopicElementDatasetFactory(TopicElementFactory):
```

Comment:
maybe not in this PR but we will need to add dataservice elements (specifically for simplifions.data.gouv.fr) to topics, and maybe organizations ?

Reply:
Like I said in https://github.com/opendatateam/udata/pull/3318/files#r2137116502, I think a good goal for this PR is to implement what was previously possible, and we can add other objects later.